workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/WorldCommandResponseHandler.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using Improbable.Gdk.Core.Commands;
+using Unity.Entities;
+using Entity = Unity.Entities.Entity;
+
+#region Diagnostic control
+
+// ReSharper disable UnusedMember.Local
+
+#endregion
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public static partial class WorldCommandsRequirables
```

Comment:
Nit - I think Command rather than Commands reads better.
Non nit - why put Requirable in the name rather than as a nested class like everything else?

Also we should really change Requirables to Requirable although that can be done later.

Reply:
nit: agreed
nit * -1: that doesn't sound too bad, will give it a try

/s/requirables/requirable/ : unsure, it's like this everywhere else so can be reconsidered later

Reply:
For removing the s I meant it should be done everywhere. So agreed it shouldn't be in this PR.